### PR TITLE
Add Apache Aurora Input Plugin

### PR DESCRIPTION
**Pull request by connorgorman**
### Required for all PRs:

- [ ] CHANGELOG.md updated (we recommend not updating this until the PR has been approved by a maintainer)
- [ ] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)
- [ ] README.md updated (if adding a new plugin)

### CODE REVIEW

File: plugins/inputs/aurora/aurora.go [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+// Description just returns a short description of the Mesos plugin
+func (a *Aurora) Description() string {
+	return "Telegraf plugin for gathering metrics from N Apache Aurora Masters"
+}
+
+func (a *Aurora) SetDefaults() {
+	if a.Timeout == 0 {
+		log.Println("I! [aurora] Missing timeout value, setting default value (1000ms)")
+		a.Timeout = 1000
```
Comment (NDonatucci) [inline plugins/inputs/aurora/aurora.go:48]: Shouldn't this be the value found in SampleConfig?

File: plugins/inputs/aurora/aurora.go [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+// Converts string values taken from aurora vars to numeric values for wavefront 
+func convertToNumeric(value string) (interface{}, bool) {
+	var err error
+	var val interface{}
+	if val, err = strconv.ParseFloat(value, 64); err == nil {
+		return val, true
+	}
+	if val, err = strconv.ParseBool(value); err != nil {
+		return val.(bool), false
```
Comment (NDonatucci) [inline plugins/inputs/aurora/aurora.go:64]: Can this assertion cause an error?

File: plugins/inputs/aurora/aurora.go [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+	}
+	return key, numeric, nil
+}
+
+// Gather() metrics from given list of Aurora Masters
+func (a *Aurora) Gather(acc telegraf.Accumulator) error {
+	a.SetDefaults()
+
+	client := http.Client{
+		Timeout: time.Duration(a.Timeout) * time.Second,
```
Comment (NDonatucci) [inline plugins/inputs/aurora/aurora.go:140]: How long is this supposed to be?

File: plugins/inputs/aurora/aurora.go (hunk 1 above)
Comment (NDonatucci) [reply plugins/inputs/aurora/aurora.go:48]: Or maybe the value in sampleConfig should be that one

File: plugins/inputs/aurora/aurora.go (hunk 3 above)
Comment (connorgorman) [reply plugins/inputs/aurora/aurora.go:140]: Good catch. That's too long

File: plugins/inputs/aurora/aurora.go (hunk 2 above)
Comment (connorgorman) [reply plugins/inputs/aurora/aurora.go:64]: Turns out I don't need the cast. Thanks

File: plugins/inputs/aurora/aurora.go (hunk 1 above)
Comment (connorgorman) [reply plugins/inputs/aurora/aurora.go:48]: Fixed to be 1000 everywhere

**Pull request merged**